src/Symfony/Component/Serializer/Debug/TraceableSerializer.php

```diff
@@ -32,6 +32,7 @@ class TraceableSerializer implements SerializerInterface, NormalizerInterface, D
     public function __construct(
         private SerializerInterface&NormalizerInterface&DenormalizerInterface&EncoderInterface&DecoderInterface $serializer,
         private SerializerDataCollector $dataCollector,
+        private readonly string $serializerName,
```

Comment:
Adding a new constructor argument is a BC break. And marking those traceable classes as `@internal` is bad, because FrameworkBundle is the one instantiating them, which relies on BC (so that updating `symfony/serializer` does not break a project still using `framework-bundle` 7.1). They should not have been marked as `@internal`

Reply:
Makes sense, fixed. I've removed the `@internal` tags as well.